docs/workflows/export-import.md

```diff
@@ -6,14 +6,30 @@ contentType: howto
 
 # Export and import workflows
 
+/// warning | Sharing credentials
+Note that the exported JSON file of your workflow will contain your credentials as well. If you don't want to share your credentials (recommended), delete them from the JSON file.
```

Comment:
This isn't fully accurate, The workflow JSON won't contain the credential but it will include the name and ID of the credential, The ID is not a sensitive value but the name the user inputs could be.

The bigger risk is potentially HTTP Request nodes that include auth headers in the node itself rather than as generic credentials, This would normally happen if you import from CURL.

Reply:
thanks for clarification, will make changes!